starboard/android/apk/app/src/main/java/dev/cobalt/coat/CobaltActivity.java

```diff
@@ -196,6 +196,10 @@ private static boolean hasArg(List<String> args, String argName) {
   protected void onNewIntent(Intent intent) {
     super.onNewIntent(intent);
     getStarboardBridge().handleDeepLink(getIntentUrlAsString(intent));
+    String url = getIntentUrlAsString(intent);
+    if(url != null ) {
+      webView.loadUrl(url);
```

Comment:
Somehow this won't do a deeplink or redirect if the app already has a page loaded.

Reply:
Idk, it sorta worked here for me. I'll merge and we have a new APK